### ISSUE TITLE
S-Meter and AM above 136MHz

### ISSUE THREAD

**Issue by Alcotan2** [piotr022/UV_K5_playground#36]
Good day. Sorry if this is not the right place to ask. You have done a great job with this walkie.

I have tried the RSSI bar mod, and I like it, but I use this other one: k5_v2.01.19_mod_AM_18to850mhz.bin, because in my city the AM signal that  that I receive best is at 136.520 MHz.

Could you make these mods capable of AM above 135.9Mhz? Rx only. Is it possible?

Thank you very much in advance for your time and your response.

**Comment by Tunas1337** [comment]
You can use the uvmod-kitchen to change Band 2 to cover above 136MHz. Band 2 is the one that has AM.

**Comment by troilus** [comment]
> You can use the uvmod-kitchen to change Band 2 to cover above 136MHz. Band 2 is the one that has AM.

If I change Band 2 to 18~1300, then there is only 1 Band on device without other bad effects? 😊

**Comment by Tunas1337** [comment]
> > You can use the uvmod-kitchen to change Band 2 to cover above 136MHz. Band 2 is the one that has AM.
> 
> If I change Band 2 to 18~1300, then there is only 1 Band on device without other bad effects? blush

Yes. The only bad effect is that any time you key in a frequency, you'll be snapped to F2 even if you'd manually changed band before, which means you can never key in a frequency for any other band, ever. Not that you'd need to.

**Comment by troilus** [comment]
Thanks, I edit the k5_v3 fw to 18~1300, but unable to set AM in sw like 29mhz, I noticed other k5_v26_18to1300 can do that, How can I change the Band, shoud I set Band 1 to 0~0 and Band 2 set to 18~108?

> > > You can use the uvmod-kitchen to change Band 2 to cover above 136MHz. Band 2 is the one that has AM.
> > 
> > 
> > If I change Band 2 to 18~1300, then there is only 1 Band on device without other bad effects? blush
> 
> Yes. The only bad effect is that any time you key in a frequency, you'll be snapped to F2 even if you'd manually changed band before, which means you can never key in a frequency for any other band, ever. Not that you'd need to.

**Comment by troilus** [comment]
> > > You can use the uvmod-kitchen to change Band 2 to cover above 136MHz. Band 2 is the one that has AM.
> > 
> > 
> > If I change Band 2 to 18~1300, then there is only 1 Band on device without other bad effects? blush
> 
> Yes. The only bad effect is that any time you key in a frequency, you'll be snapped to F2 even if you'd manually changed band before, which means you can never key in a frequency for any other band, ever. Not that you'd need to.

I check the k5_v26 18to1300 find it is  F1:50-76 F2:18-135.9

**Comment by Alcotan2** [comment]
> You can use the uvmod-kitchen to change Band 2 to cover above 136MHz. Band 2 is the one that has AM.



Thank you very much. I got it. I had some troubles becasuse in Build.bat I had to substitute "python" by "py". And then in the mod_custom_freq_ranges.py, I had to substitute lines  "if...." with the ranges of the firmware. It was first time, but I have learned how uvmod_kitchen works. 

Thanks again.
